heron/scheduler-core/tests/java/com/twitter/heron/scheduler/dryrun/testdata/SubmitDryRunOutputATable.txt

```diff
@@ -0,0 +1,20 @@
+[1mContainer 1[0m
```

Comment:
I was following the existing structure:
![image](https://cloud.githubusercontent.com/assets/3656079/22040347/60ea7f86-dcb7-11e6-9b5d-ad1ebc64462c.png)